services/api/db/utils.py

```diff
@@ -0,0 +1,31 @@
+from django.core.mail import EmailMultiAlternatives, get_connection
+from django.template.loader import render_to_string
+from django.utils.html import strip_tags
+from django.conf import settings
+
+
+#TODO: This file will be used for various common email tasks.
+def send_reset_password_email(recipient_email, reset_url):
+    try:
+        connection = get_connection(
+        backend=settings.EMAIL_BACKEND,
+        host=settings.EMAIL_HOST,
+        port=settings.EMAIL_PORT,
+        username=settings.EMAIL_HOST_USER,
+        password=settings.EMAIL_HOST_PASSWORD,
+        use_tls=True, ) # or use_ssl=True for SSL
+        subject = "Reset Your Password on"
```

Comment:
this subject text seems to be incomplete

Reply:
changed it